src/ctapipe/calib/camera/extractor.py

```diff
@@ -41,18 +37,24 @@ def __call__(
         col_name="image",
     ) -> list:
         """
-        Prepare the extraction chunks and call the relevant function of the particular extractor
-        to extract the statistical values.
+        Divides the input DL1 table into overlapping or non-overlapping chunks of size `chunk_size`
+        and call the relevant function of the particular extractor to extract the statistical values.
+        The chunks are generated in a way that ensures they do not overflow the bounds of the table.
+        - If `chunk_shift` is None, extraction chunks will not overlap, but the last chunk is ensured to be
+        of size `chunk_size`, even if it means the last two chunks will overlap.
+        - If `chunk_shift` is provided, it will determine the number of samples to shift between the start
```

Comment:
Do we allow `chunk_shift > chunk_size` resulting in non-contiguous chunks?

Reply:
good point. We should not allow this scenario.